client/src/pages/course/admin/mentors.tsx

```diff
@@ -75,6 +75,19 @@ function Page(props: CoursePageProps) {
     }
   };
 
+  const handleRestore = async ({ githubId }: MentorDetails) => {
+    try {
+      setLoading(true);
+      await service.restoreMentor(githubId);
+      const records = mentors.map(r => (r.githubId === githubId ? { ...r, isActive: true } : r));
+      setMentors(records);
```

Comment:
You cannot rely on "mentors" state.
Please the next form: `setMentors(prevState => /* do something with prevState */)`.
https://reactjs.org/docs/hooks-reference.html#usestate

Reply:
Done